libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -628,78 +668,37 @@ static IotMqttError_t _subscriptionCommon( IotMqttOperationType_t operation,
     {
         EMPTY_ELSE_MARKER;
     }
+    IOT_FUNCTION_CLEANUP_BEGIN();
```

Comment:
`IOT_FUNCTION_EXIT_NO_CLEANUP()` should be used if the cleanup section is empty.

Reply:
Changed